RFC/src/RFC-0201_TariScript.md

```diff
@@ -287,65 +285,69 @@ The updated input definition is:
 pub struct TransactionInput {
     /// Options for an output's structure or use
     features: OutputFeatures,
-    /// The homomorphic commitment representing the output amount
+    /// The homomorphic Pedersen commitment representing the output amount
     commitment: Commitment,
-    
     /// The serialised script
     script: Vec<u8>,
     /// The script input data, if any
-    input_data: Vec<u8>
-    /// The block height that the UTXO was mined 
-    height : u64
-    /// A signature with k_s, signing the script, input data, and mined height
-    script_signature: Signature,
-    /// The offset pubkey, K_O
+    input_data: Vec<u8>,
+    /// Signature signing the script, input data, public script key and the homomorphic commitment of the TransactionInput with
+    /// a combination of the homomorphic commitment private values (amount and blinding factor) and private script key.
+    script_signature: CommitmentSignature,
+    /// The script offset pubkey, K_O
     script_offset_public_key: PublicKey
 }
 ```
 
-The input data to the script is signed with resolving script public key \\(K_s \\), which proves that the spender provides
-the input to the script.
-
-The `height` field is the height this UTXO was mined at. This is to stop [Replay attacks](#replay-attacks).
-
-The `script_signature` is a Schnorr signature. It signs the script, the script input, and the height the UTXO was originally mined:
+The `script_signature` is an aggregated Schnorr signature signed with a combination of the homomorphic commitment private values \\( (v\_i \\, , \\, k\_i )\\) and private script key \\(k\_{Si}\\) to prove ownership of thereof, see [Signature on Commitment values] by F. Zhang et. al. and [Commitment Signature] by G. Yu. It signs the script, the script input, _public script key_ and the commitment:
 
 $$
-  s_{Si} = r_{Si} + \hash{ R_{Si} \cat \alpha_i \cat \input_i \cat h_i} k_{Si}
-$$ 
-
-### Consensus changes
-
-The Mimblewimble balance for blocks and transactions stays the same.
+\begin{aligned}
+ s_{Si} = (s_{S1i}, s_{S2i}, R_{Si} )
```

Comment:
Nit: This is a readability nit, for the two commitment components of the CommSig I think perhaps use the u and v  notation from the paper as it is more readable than S1 and S2. There are already a lot of S's and i's in the notation and u and v will stand out better. 
```suggestion
 s_{Si} = (u_{Si}, v_{Si}, R_{Si} )
```